### PR TITLE
Update dependencies and travis configuration

### PR DESCRIPTION
**Pull request by vperron**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "acorn-stage3": "^2.0.0",
     "acorn-walk": "^6.2.0",
     "cheerio": "^1.0.0-rc.3",
+    "codecov": "^3.6.5",
```
Comment (janlazo) [inline package.json]: Why is this a dependency instead of devDependency?

Comment (vperron) [reply package.json]: That's indeed an issue, it should not.

Comment (vperron) [reply package.json]: Can you fix this maybe ?

Comment (janlazo) [reply package.json]: https://github.com/Polyconseil/easygettext/pull/78

**Pull request merged**